app/views/layouts/design_system.html.erb

```diff
@@ -15,6 +15,32 @@
       navigation_items: navigation_links_internal + navigation_links_auth,
   } %>
 
+  <% unless flash.empty? %>
+    <div class="govuk-width-container">
+      <div class="govuk-grid-row app-flash-container">
+        <% flash.each do |type, msg| %>
+          <% if type == "notice" %>
+            <%= render "govuk_publishing_components/components/notice", {
+              description: msg,
+              margin_bottom: 0,
+            } %>
+          <% elsif type == "success" %>
+            <%= render "govuk_publishing_components/components/success_alert", {
+              message: msg,
+              margin_bottom: 0,
+            } %>
+          <% elsif type == "error" %>
+            <%= render "govuk_publishing_components/components/error_alert", {
+              message: msg,
+            } %>
+          <% else %>
+            <%= msg %>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+  <% end %>
+
```

Comment:
Would it make any sense to move this out in to a separate commit? It doesn't seem directly related to the show page

Reply:
It's tricky, because the show page is the first existing page moving over to the design system. The show page uses flash messages, so it is what's driving the need for this being added to the layout template.

Reply:
If that's what rubocop wants, that's good enough for me 🙈